### PR TITLE
Create rojan.json

### PR DESCRIPTION
**Pull request by RojanGamingYT**
### Requirements
- [x] You're not using Vercel or Netlify.
- [x] You have completed your website, there's no type of placeholder at the website. *This requirement is optional if you are registering a domain for emails.*
- [x] The website is reachable.
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.
   - *You should have your email presented. If you don't want to share email, you can leave email an empty string (`""`) and add any other social such as Discord/Twitter/etc.*

### Link to Website

https://rojangamingyt.github.io/

### CODE REVIEW

File: domains/rojan.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "repo": "https://github.com/RojanGamingYT/is-a-dev-portfolio",
+    "owner": {
+        "username": "RojanGamingYT",
+        "email": "email.rojan@gmail.com",
+        "discord": "RojanGamingYT#4466"
+    },
+    "record": {
+        "CNAME": "rojangamingyt.github.io"
```
Comment (WilliamDavidHarrison) [inline domains/rojan.json:9]: 404?

**Review by WilliamDavidHarrison — CHANGES_REQUESTED** [review]

**Comment by RojanGamingYT** [comment]
> question, since link to website is .../is-a-dev-portfolio, your website won't be accessible in https://rojan.is-a.dev but https://rojan.is-a.dev/is-a-dev-portfolio, confirm?

I want it accessible in https://rojan.is-a.dev

**Comment by RojanGamingYT** [comment]
> > > question, since link to website is .../is-a-dev-portfolio, your website won't be accessible in https://rojan.is-a.dev but https://rojan.is-a.dev/is-a-dev-portfolio, confirm?
> > 
> > 
> > I want it accessible in https://rojan.is-a.dev
> 
> uuuuuuhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhh then i don't think that it's possible you can try rename the repository containing ur files to rojangamingyt.github.io and then remove the part after the slash in the CNAME record

I have fixed it and its now available at https://rojangamingyt.github.io/

**Review by WilliamDavidHarrison — APPROVED** [review]

**Comment by RojanGamingYT** [comment]
Why isnt it merged?

**Pull request closed without merging**